rpcs3/CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ else()
 	set(ADDITIONAL_LIBS "")
 endif()
 
+if(UNIX)
```

Comment:
I'm not sure whether to add `AND NOT APPLE` because a few lines above there's `-I/opt/X11/include` but rpcs3/rpcs3.cpp has `&& !defined(__APPLE__)`.